src/components/views/TrackView.js

```diff
@@ -7,33 +7,8 @@ import TrackTimeline from "../common/TrackTimeline";
 
 class TrackView extends Component {
     render() {
-      const {judgments} = this.props;
       const {search} = this.props;
-      const {raceGroups} = this.props;
-      const {stage} = this.props;
-
-
-      let elements = [];
-      judgments.map(judgment => {elements.push({distance : judgment.distance, text : judgment.name})});
-      raceGroups.map(raceGroup => {elements.push({distance : 90, text : raceGroup.raceGroupType})});
-      elements.sort((a,b) => b.distance - a.distance);
-      let elems = [];
-      let lastElement = null;
-      let size = elements.length;
-      let sizeBottom = 0;
-      elements.map((elem, i) => {
-          let gap = 0;
-          if (i === 0) {
-              gap = stage.distance - elem.distance;
-          } else if(size === i + 1) {
-              sizeBottom = elem.distance;
-            gap = lastElement.distance - elem.distance;
-          } else {
-              gap = lastElement.distance - elem.distance;
-          }
-          lastElement = elem;
-          elems.push({distance : elem.distance, text : elem.text, gap: gap});
-      });
+      const {timeline} = this.props;
 
       return(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "stage" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'stage' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AUnusedVariable)